src/main/resources/templates/pollingunitsreact.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:th="http://www.thymeleaf.org">
+<head>
+    <!--Import Google Icon Font-->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <!--Import materialize.css-->
+    <link type="text/css" rel="stylesheet" th:href="@{css/materialize.min.css}"  media="screen,projection"/>
+
+    <!--Let browser know website is optimized for mobile-->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+        
+    <link rel="stylesheet" type="text/css" th:href="@{/css/style.css}"/>
+	<body>
+    <h2>Polling Units</h2>
+        <div id="react">
+            <!-- Global site tag (gtag.js) - Google Analytics -->
+            <script async src="https://www.googletagmanager.com/gtag/js?id=UA-141725675-1"></script>
+            <script type="text/javascript" th:src="@{/js/googleanalytics.js}"></script>
+            </head>
+            <!--Start of Tawk.to Script-->
+            <script type="text/javascript" th:src="@{/js/tawkto.js}"></script>
+            <!--End of Tawk.to Script-->
+            <!--JavaScript at end of body for optimized loading-->
+            <script type="text/javascript" th:src="@{/js/materialize.min.js}"></script>
+
+            <!-- Load our React component. -->
+            <script th:src="@{/built/bundle.js}"></script>
```

Comment:
@dinobi @benfluleck This html page is the basic react version that displays a table of records. Not yet pageable and sortable.  Goal is to also introduce full-text search and column filtering.  Deployed at https://naijagov.herokuapp.com/pollingunitsreact